Sat May 30 12:32:37 2020 User admin logged in
Sat May 30 12:33:06 2020 User admin payed the order Order for admin
Sat May 30 12:33:12 2020 User admin logged out
Internal Server Error: /api/games/
Traceback (most recent call last):
  File "/home/sleeping/.local/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/sleeping/.local/lib/python3.5/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/sleeping/.local/lib/python3.5/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/sleeping/lic_shop/shop/views.py", line 179, in dump_games
    return JsonResponse({'games': games})
  File "/home/sleeping/.local/lib/python3.5/site-packages/django/http/response.py", line 558, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "/usr/lib/python3.5/json/__init__.py", line 237, in dumps
    **kw).encode(obj)
  File "/usr/lib/python3.5/json/encoder.py", line 198, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.5/json/encoder.py", line 256, in iterencode
    return _iterencode(o, 0)
  File "/home/sleeping/.local/lib/python3.5/site-packages/django/core/serializers/json.py", line 104, in default
    return super().default(o)
  File "/usr/lib/python3.5/json/encoder.py", line 179, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <QuerySet [<Game: Test Game>, <Game: foobaaar>]> is not JSON serializable
Internal Server Error: /api/games/
Traceback (most recent call last):
  File "/home/sleeping/.local/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/sleeping/.local/lib/python3.5/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/sleeping/.local/lib/python3.5/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/sleeping/lic_shop/shop/views.py", line 187, in dump_games
    return JsonResponse({'games': games_to_json(games)})
  File "/home/sleeping/lic_shop/shop/views.py", line 181, in games_to_json
    'game_id': g.id} for g in games]
TypeError: 'Game' object is not iterable
Sat May 30 14:04:52 2020 User admin logged in
Sat May 30 14:05:00 2020 User admin payed the order 6
Not Found: /favicon.ico
Sat May 30 21:40:51 2020 Registered new user chernyikwadrat
Sat May 30 21:40:51 2020 User chernyikwadrat logged in
Sat May 30 21:41:10 2020 User chernyikwadrat payed the order 7
Sat May 30 21:59:47 2020 User admin payed the order 8
Tue Jun  2 23:06:47 2020 User admin logged out
Tue Jun  2 23:06:55 2020 User chernyikwadrat logged in
Tue Jun  2 23:07:05 2020 User chernyikwadrat payed the order 9
Thu Jun  4 01:16:13 2020 User chernyikwadrat logged out
Not Found: /favicon.ico
Thu Jun  4 21:24:33 2020 User admin payed the order 10
Thu Jun  4 22:33:28 2020 User admin logged out
Thu Jun  4 22:34:34 2020 Registered new user mirinda
Thu Jun  4 22:34:34 2020 User mirinda logged in
Thu Jun  4 22:42:55 2020 User admin logged out
Thu Jun  4 22:43:03 2020 User mirinda logged in
Thu Jun  4 22:47:00 2020 User mirinda payed the order 11
Thu Jun  4 23:27:10 2020 User admin logged out
Thu Jun  4 23:27:35 2020 User admin logged out
Thu Jun  4 23:27:43 2020 User mirinda logged in
Fri Jun  5 17:59:59 2020 User admin logged in
Fri Jun  5 18:00:16 2020 User admin logged out
Fri Jun  5 18:56:32 2020 Registered new user kmm
Fri Jun  5 18:56:32 2020 User kmm logged in
Fri Jun  5 19:01:19 2020 User kmm logged out
Fri Jun  5 19:01:35 2020 User kmm logged in
